recipes-support/libgpg-error/libgpg-error_%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append_qemuriscv32 = "\
```

Comment:
this should be ```riscv32``` so it can apply to all machines and not just to qemu

Reply:
Fixed in updated patch